src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,116 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int DEFAULT_GROWN_FACTOR = 2;
+    private static final double DEFAULT_LOAD_FACTOR = 0.75;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
```

Comment:
Use the diamond operator to avoid raw types like 'new Node[DEFAULT_CAPACITY]'. It should be 'new Node<>(DEFAULT_CAPACITY)'. This prevents potential ClassCastException issues.